extensions/smallrye-reactive-messaging/deployment/src/test/java/io/quarkus/smallrye/reactivemessaging/config/ConnectorConfigSupportsEnvVarOverridesTest.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.smallrye.reactivemessaging.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junitpioneer.jupiter.SetEnvironmentVariable;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+@SetEnvironmentVariable(key = "MP_MESSAGING_INCOMING_WAY_IN_TRACING_ENABLED", value = "false")
```

Comment:
Wasn't that an issue on Java 14+ because it uses an internal API and needs to be opened explicitly?